include/picongpu/param/species.param

```diff
@@ -38,12 +38,23 @@ namespace picongpu
 {
 
 /** Particle Shape definitions
- *  - particles::shapes::CIC : 1st order
- *  - particles::shapes::TSC : 2nd order
- *  - particles::shapes::PCS : 3rd order
- *  - particles::shapes::P4S : 4th order
- *
- *  example: using UsedParticleShape = particles::shapes::CIC;
+ *  Particle shape naming is according to
+ *    Hockney R, Eastwood JW. Computer simulation using particles. IOP Pubslishing Ltd (1988). p. 144.
+ *  That is, the name of a particle shape is defined by the visual form of its cloud shape S,
+ *  which describes a particle's charge density distribution.
+ *  However, the charge assigned to a mesh point due to a distribution of overlapping finite-sized
+ *  particles is obtained by evaluating and summing the assignment function W for all particles.
+ *  The assignment function is obtained by convolving the cloud shape with the top-hat function.
+ *  PIConGPU only implements the assignment functions.
+ *  **NOTE, the higher order shape names are redefined with release 0.6.0 in order to provide a consistent naming:
+ *      -> PQS is the name of the 3rd order shape (instead of PCS)
+ *      -> PCS is the name of the 4th order shape (instead of P4S)
+ *      -> P4S does not exist anymore
+ *  **
+ *  - particles::shapes::CIC : 1st order, assignment function is a piecewise linear spline
+ *  - particles::shapes::TSC : 2nd order, assignment function is a piecewise quadratic spline
+ *  - particles::shapes::PQS : 3rd order, assignment function is a piecewise cubic spline
+ *  - particles::shapes::PCS : 4th order, assignment function is a piecewise quartic spline
```

Comment:
What does the `order` part of the comment add? The information on the assignment function is given after the comma. While out of context this `order` may be misunderstood as the cloud density order.

Reply:
Well, nothing anymore. Just didn't remove it. Will do now.